src/main/webapp/js/components/settings/app-settings.jsx

```diff
@@ -0,0 +1,83 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import _ from 'lodash';
+import { Form } from 'react-final-form';
+import Alert from 'react-s-alert';
+import { withRouter } from 'react-router-dom';
+
+import renderFormField from '../../utils/form/form-utils';
+import { updateEntity, fetchEntity } from '../../actions/entity-actions';
```

Comment:
It's not a typical entity, you don't need an id for it, it will be best to write separate methods to get and save the settings